spec/log-window-tests.js

```diff
@@ -26,20 +26,33 @@ describe('Log window', () => {
     mockDocument.createElement = (name) => document.createElement(name);
     document.body.appendChild(mockDocument);
 
+    chrome = {
+      windows: {
+        create: () => { },
+        update: () => { },
+      },
+      runtime: {
+        getURL: () => { },
+      }
+    }
     mockWindow = {
-      closed: false,
-      focus: () => {},
-      document: mockDocument,
+      id: 1234567,
     };
+
+    // Return the mock window when we are supposed to create one.
+    spyOn(chrome.windows, 'create').and.returnValue(
+      new Promise((resolve, reject) => {
+        resolve(mockWindow)
+      }));
+
+    spyOn(chrome.windows, 'update').and.callThrough();
```

Comment:
I don't think you need to callThrough() if there's nothing there.

Reply:
Gotcha! Removed this line. Thanks